### PR TITLE
Doc workflow improvements

### PR DESCRIPTION
**Pull request by CamDavidsonPilon**
Doc improvements - take a look at the commits for a breakdown of changes. 

The biggest change you'll notice is including all doc pages here. This is better for tracking doc changes. Maybe we remove the generated folder prefix as well. 

I cleaned up sphinx and latex problems - sometimes using a global search and replace, so there may be errors in that. I didn’t touch any of the main docs content here.

### CODE REVIEW

**Comment by CamDavidsonPilon** [comment]
Oh, this isn’t pointing at your latest branch, so there may be conflicts with it.

File: zepid/causal/gformula/TimeVary.py [hunk 1]
```diff
@@ -1055,77 +1062,78 @@ def _predict(df, model, variable):
 
 
 class IterativeCondGFormula:
+    """Time-varying implementation of the g-formula, also referred to as the g-computation algorithm formula. The
```
Comment (pzivich) [inline zepid/causal/gformula/TimeVary.py]: Same as previous comment (does it need `r"""`?)

File: zepid/causal/ipw/IPMW.py [hunk 2]
```diff
@@ -5,81 +5,86 @@
 
 
 class IPMW:
+    """Calculates the weight for inverse probability of missing weights using logistic regression. `IPMW`
```
Comment (pzivich) [inline zepid/causal/ipw/IPMW.py]: `r"""`?

File: zepid/sensitivity_analysis/Simple.py [hunk 3]
```diff
@@ -5,41 +5,44 @@
 
 
 class MonteCarloRR:
+    """Monte Carlo simulation to assess the impact of an unmeasured binary confounder on the results
```
Comment (pzivich) [inline zepid/sensitivity_analysis/Simple.py]: `r"""`?

**Comment by pzivich** [comment]
@CamDavidsonPilon No worries. You can keep it pointed at `master`. I think these changes will be done before I finish 0.6.0. I can resolve the changes when I merge #81

File: zepid/sensitivity_analysis/Simple.py (hunk 3 above)
Comment (CamDavidsonPilon) [reply zepid/sensitivity_analysis/Simple.py]: we only need `r` if there is some python escape sequence, like `\f` (which shows up in latex)

File: zepid/calc/utils.py [hunk 4]
```diff
@@ -1115,12 +1166,13 @@ def screening_cost_analyzer(cost_miss_case, cost_false_pos, prevalence, sensitiv
...
-    >>>from zepid.calc import screening_cost_analyzer
-    >>>screening_cost_analyzer(cost_miss_case=1, cost_false_pos=3, prevalence=0.15, sensitivity=0.9, specificity=0.88)
+
+    >>> from zepid.calc import screening_cost_analyzer
+    >>> screening_cost_analyzer(cost_miss_case=1, cost_false_pos=3, prevalence=0.15, sensitivity=0.9, specificity=0.88)
     """
     print('----------------------------------------------------------------------')
-    print('''NOTE: When calculating costs, be sure to consult experts in health\npolicy or related fields.  
-        Costs should encompass more than only monetary\ncosts, like relative costs (regret, disappointment, stigma, 
+    print('''NOTE: When calculating costs, be sure to consult experts in health\npolicy or related fields.
```
[the remark this answers is not in the document]
Comment (CamDavidsonPilon) [reply zepid/calc/utils.py]: hrmmm, well, with a warning it can be suppressed if the user wants, so yea 👍

**Comment by pzivich** [comment]
I updated the logo on the subpages. Everything lgtm otherwise. Feel free to merge to master

Thanks again for all your help with ReadTheDocs!!

File: zepid/causal/doublyrobust/TMLE.py [hunk 5]
```diff
@@ -626,8 +627,8 @@ def _bounding(v, bounds):
     @staticmethod
     def _unit_bounds(y, mini, maxi, bound):
         v = (y - mini) / (maxi - mini)
-        v = np.where(np.isnan(v), np.nan, np.where(v < bound, bound, v))
-        v = np.where(np.isnan(v), np.nan, np.where(v > 1-bound, 1-bound, v))
+        v = np.where(np.less(v, bound), bound, v)
+        v = np.where(np.greater(v, 1-bound), 1-bound, v)
```
Comment (CamDavidsonPilon) [inline zepid/causal/doublyrobust/TMLE.py]: TIL

**Pull request merged**